specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2021-08-08/AlertProcessingRules.json

```diff
@@ -555,7 +560,11 @@
           "description": "List of recurrences.",
           "items": {
             "$ref": "#/definitions/Recurrence"
-          }
+          },
+          "x-ms-identifiers": [
+            "recurrenceType",
```

Comment:
Neither of these, nor the combination is unique. 
StartTime is optional.  I think this should be empty.